decriptive_stats.R

```diff
@@ -0,0 +1,51 @@
+
+descriptive_stats <- function(data, rem_na = NULL) {
+    # data is a dataframe
+    # rem_na = 'TRUE' or 'FALSE'. The default is "TRUE"
+    
```

Comment:
I tested this using `descriptive_stats(most_recent_measurement_of_interest_df %>% select_if(is.numeric))`. Looks good!

Recommend you do that `select_if` inside the function so that it will be tolerant of dataframes with date-valued columns, etc..